File: yunu/프로그래머스/숫자 게임/내 코드.js
function solution(A, B) {
  A.sort((a, b) => b - a);
  B.sort((a, b) => b - a);

  let j = 0;
  for (let i = 0; i < A.length; i++) {
    if (A[i] < B[j]) {
      j++;
    }
  }

  return j;
}

File: yunu/프로그래머스/연속된 부분 수열의 합/내 코드.py
# https://freedeveloper.tistory.com/393 참고
def solution(sequence, k):
    n = len(sequence)
    start = 0
    end = 0
    seqSum = 0
    seqArr = []

    for start in range(n):
        while seqSum < k and end < n:
            seqSum += sequence[end]
            end += 1

        if seqSum == k:
            seqArr.append([start, end - 1])

        seqSum -= sequence[start]

    return sorted(seqArr, key=lambda seq: seq[1] - seq[0])[0]